src/tester.tsx

```diff
@@ -44,22 +47,26 @@ const NullComponent = (props) => <Fragment {...props} />;
  * @returns {Tester}
  */
 class Tester {
-  config;
-  initialMount;
-  onBeforeMount;
-  opts;
-  profile;
-  props;
-  shallow;
-  TestedComponent;
-  wrapper;
-  wrappers;
-
-  constructor (TestedComponent, opts = {}) {
-    this.config = this.constructor.Configuration;
-    this.opts = opts;
+  static Configuration: ConfigurationClass;
+
+  public opts: ITesterOps;
+
+  public config: ConfigurationClass;
+  public initialMount: React.ReactNode;
+  public onBeforeMount?: (tester: Tester) => Promise<void>;
+  public profile: IProfile;
+  public props: object;
+  public TestedComponent: ComponentClass;
+
+  public AppState: any;
+  public shallow: any;
+  public wrapper: any;
+
+  constructor (TestedComponent: ComponentClass, opts: ITesterOps = {}) {
+    this.config = Tester.Configuration;
```

Comment:
Are you sure that this is really working ?
I'd double check ...

Reply:
No, but I saw it in an example, and the tests pass. I thing the best thing would be to just try upgrading one of our apps and see what breaks.

Reply:
Sounds good to me.